### ISSUE TITLE
Streaming causes the image to be fouled up beyond all recognition

### ISSUE THREAD

**Issue by ScreamingDev** [thestr4ng3r/chiaki#292]
**Environment**
 - OS/Distribution: Ubuntu 19.10
 - Desktop Environment: xfce4
 - Hardware: intel/nvidia
 - Exact Version or Commit Hash and Installation Method: 1.2.1

**Describe the bug**

The image heavily clutters when OBS 25.0.8 begins streaming.

**To Reproduce**

1. Connect to PS4 with 720p @30 fps and leaving Bitrate and Audio Buffer Size empty.
2. Set OBS to stream the Chiaki window

(until here the image is clear and without clutter even when moving around - especially when setting the Bitrate to 5120)

3. Start streaming
4. Start moving around
5. See how the image falls apart (in OBS and live in stream)

*Hint: Recording (in OBS) does not harm the image and all records are clean but streaming makes the image fall apart*

**Expected behavior**

The image has no clutter.

**Log Files**

[chiaki_session_2020-08-01_18-33-16-020020.log](https://github.com/thestr4ng3r/chiaki/files/5010963/chiaki_session_2020-08-01_18-33-16-020020.log)

**Screenshots**

![image](https://user-images.githubusercontent.com/2559177/89106256-a88fd400-d428-11ea-86aa-d11378988271.png)

**Additional context**

**Comment by thestr4ng3r** [comment]
Judging from the logs, only a fraction of UDP packets gets through. It seems like OBS is taking up all your network resources. Are you streaming over wifi or is your PS4 maybe connected over the internet?

**Comment by ScreamingDev** [comment]
Thanks for the fast reply :v: 
PS4 is in Wifi, Laptop is connected via cable (same network)
Unfortunately I need to buy another cable and can reply to this on Monday, when both are connected via cable.
Hope that does it.

**Comment by ScreamingDev** [comment]
To me this issue is done and I like to let you know how/why:

TL;DR Chiaki is an absolutely nice thing to use and the surroundings (like Network, Codecs etc) need improvements.

--- 

For the last weeks I fiddled around with the network and the more I investigated the better it got but I never achieved a high quality. 

- After shutting down every device and application that causes traffic the image stayed blurry as it is.
- So I connected the PS4 and the computer via cable to the router, which allowed a bit more stable image but not much.
- The clutter comes back during action scenes on the screen which got a bit better after connecting PS4 and PC via cable directly to the modem - obviously my modem can sort packages better than my router LEL
- I tried some buffer sizes which lowered the clutter but also the image quality.

After all the problem is not 100% sorted out but Chiaki is an absolutely nice application for streaming the PS4 image to the PC. I am hoping that some codecs and compression gets better but that's not in the hand of Chiaki I guess. I'll continue looking at Wireshark and try to tcpdump on the router/modem because I would rather use Chiaki than a capture card because I also want to play via VPN from anywhere ^^
